package.json

```diff
@@ -69,13 +71,16 @@
     "lint-staged": "10.5.1",
     "mongoose": "5.10.9",
     "mysql": "2.18.1",
+    "mysql2": "2.2.5",
```

Comment:
What is this for?

Reply:
For Sequelize to work as written in the NestJS documentation --> https://docs.nestjs.com/techniques/database#sequelize-integration

Reply:
Ah gotcha, nice!